community.html

```diff
@@ -20,7 +20,7 @@ <h2>Connect with us</h2>
   <ul id="socialmedia">
     <li><a class="fediverse" href="https://floss.social/@sandstorm">Fediverse</a>
     <li><a class="github" href="https://github.com/sandstorm-io/sandstorm">GitHub</a>
-    <li><a class="livechat" href="https://kiwiirc.com/client/irc.libera.chat/?channel=#sandstorm">Live Chat</a>
+    <li><a class="livechat" href="https://matrix.to/#/#sandstorm:libera.chat">Live Chat</a>

```

Comment:
I sort of think we should change Live Chat up here to say Matrix. It's unclear what the M logo is otherwise to people who don't recognize it, and "Live Chat" feels shockingly generic next to "Fediverse", "GitHub", and "YouTube". (I am sort of fine not mentioning the Google Groups specifically for the mailing list... it's the next thing I want to replace anyways.)

Reply:
I'm just a little worried that "Matrix" is not totally unambiguous. That's also why I say "Matrix chat". (Though in one case I say chat twice which I'll definitely remove)

Reply:
My initial hope was that the irc logo was an actual picture and I could just erase the letters and change the word to "Matrix" as you say. Maybe I could still do that. I could add a speech bubble around the M logo.

Reply:
With a speech bubble, I wouldn't worry about Matrix being ambiguous.

Also "Fediverse" is kind of generic.

Reply:
Well my latest version kind of sucks but that's an idea I can try to improve. What do you think though?